premake5.lua

```diff
@@ -27,6 +27,8 @@
 		trigger = "embed",
 		description = "Embed scripts in scripts.c; required before release builds",
 		execute = function ()
+			_MAIN_SCRIPT_DIR = os.getcwd()
+			_SCRIPT_DIR = path.join(_MAIN_SCRIPT_DIR, "scripts")
```

Comment:
I had to add those lines to premake4.lua because the 4.x code doesn't set those variables. Premake 5.x does, so these shouldn't be needed. You might want to try again without them, or provide more information about what's going wrong so we can figure out the right fix.